Don't surround MYPAINT_BRUSHES_DIR with escaped quotation marks.

Fixes this warning when building MyPaint:

<command line>:5:30: warning: missing terminating '"' character [-Winvalid-pp-token]

See: https://github.com/mypaint/mypaint-brushes/issues/5
--- pkgconfig.pc.in.orig	2018-12-08 13:03:53.000000000 -0600
+++ pkgconfig.pc.in	2020-06-02 21:00:18.000000000 -0500
@@ -7,4 +7,4 @@
 Description: Brushes used by MyPaint and other software using libmypaint.
 Version: @VERSION@
 
-Cflags: -DMYPAINT_BRUSHES_DIR="\"${brushesdir}\""
+Cflags: -DMYPAINT_BRUSHES_DIR="${brushesdir}"
